cmd/up/activate.go

```diff
@@ -250,9 +250,11 @@ func (up *upContext) devMode(ctx context.Context, app apps.App, create bool) err
 }
 
 func (up *upContext) createDevContainer(ctx context.Context, app apps.App, create bool) error {
-	oktetoLog.Spinner("Activating your development container...")
-	oktetoLog.StartSpinner()
-	defer oktetoLog.StopSpinner()
+	if !up.Dev.IsHybridModeEnabled() {
```

Comment:
I think we are not showing anything here if it's hybrid which can cause some friction with the user. Should we show something to let the user know that okteto is still running?

Reply:
What would you show?

Reply:
I'd show the same message as we have for sync, We are activating their development container by getting the env vars, but I don't have a strong opinion on the message we should be showing. WDYT?

Reply:
In the issue description, it is stated that we do not display this message. In my opinion, even if we activate the dev container, I think it can be confusing for the user to see this message if we are in `sync` mode. On the other hand, if we don't show a spinner at this point, we could leave the user with no feedback on how the command startup process is going (plus the activation of the dev container is the most time-consuming, I think). We could show something like `preparing development environment` or something like that? WDYT?

Reply:
👍🏻  ok, i've updated the PR to add this message, also updated the video at the description